### PR TITLE
🚨 Update login tests [#daph1d]

### PR DESCRIPTION
**Pull request by mel-OR**

### CODE REVIEW

File: pages/LoginPage.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
 class LoginPage {
   //LOCATORS
   public get loginButton() {
-    return $("#login");
+    return $("button[id='login']");
```
Comment (ckknight) [inline pages/LoginPage.ts:4]: ❓ why this change? Also, why not `button#login`?

File: specs/login.ts [hunk 2]
```diff
@@ -1,11 +1,21 @@
+import { convertToObject } from "node_modules/typescript/lib/typescript";
```
Comment (ckknight) [inline specs/login.ts:1]: 🔧
```suggestion
```

**Review by ckknight — CHANGES_REQUESTED** [review]
I believe this project might not have its eslint config set up properly. (that does not need to be fixed within this PR)

**Comment by hkolsen** [comment]
❓ @ckknight were you able to run the tests locally? I can get as far the browser navigating past Cloudflare access to `https://depmub.com/app/login` but the page never fully loads (white screen). Curious if it was something that can be reproduced since @mel-OR can't.

**Comment by ckknight** [comment]
@hkolsen no, I was only reviewing code quality, not how the execution performed.
